packages/calcite-components/src/components/block/block.stories.ts

```diff
@@ -235,6 +235,40 @@ export const loadingWithNoStatusNorSlottedIcon_TestOnly = (): string =>
     </calcite-block>
   `;
 
+export const longWrappingTextInBlockAndBlockSection_TestOnly = (): string =>
+  html`
+    <calcite-panel style="width:250px">
+      <calcite-block
+        collapsible
+        open
+        heading="Planes, trains, and automobiles are some examples of modes of transportation"
```

Comment:
Might be nice in the future if we had a JSON file with sample content like this so all of our storybook screenshot tests could be consistent with good content. Maybe JSON file with `paragraph`, `sentence`, `snippet`, `heading`, etc. Could be something as a "good first issue" to cleanup.

Reply:
Agreed `sampleText.long`, `sampleText.short` would be nice helpers to be able to reference. 

Or maybe a helper to generate "esri ipsum" with geography terms for a given character count... `getSampleText(400)`